packages/flutter_tools/build.yaml

```diff
@@ -1,5 +1,8 @@
 targets:
   $default:
+    builders:
```

Comment:
Because we use build_runner for running tests and build_runner scans the pubspec for dependencies to determine which targets to build. Since we now depend on build_web_compilers, it tries to initialize a bunch of web builders for the test script. This hasn't fully stopped the errors, I suspect we need to disable more builders.